prost-build/src/module.rs

```diff
@@ -40,6 +39,15 @@ impl Module {
         self.components.iter().map(|s| s.as_str())
     }
 
+    #[must_use]
+    #[inline(always)]
+    pub fn starts_with(&self, needle: &[String]) -> bool
```

Comment:
I believe this function should be `pub(crate)`. Similar to the function you removed.

Reply:
Done 👍🏽